api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java

```diff
@@ -446,4 +507,168 @@ private CohortDefinition getPatientsRegisteredInFichaAPSSPP() {
+    map.put(
+        "counseledOnSideEffectsOfArtConcept",
+        hivMetadata.getCounseledOnSideEffectsOfArtConcept().getConceptId());
+    map.put(
+        "adherenceEvaluationConcept", hivMetadata.getAdherenceEvaluationConcept().getConceptId());
+    map.put("goodConcept", hivMetadata.getPatientIsDead().getConceptId());
+    map.put("arvAdherenceRiskConcept", hivMetadata.getArvAdherenceRiskConcept().getConceptId());
+    map.put("badConcept", hivMetadata.getBadConcept().getConceptId());
+    map.put("artProgram", hivMetadata.getARTProgram().getProgramId());
+    map.put(
+        "transferredFromOtherHealthFacilityWorkflowState",
+        hivMetadata
+            .getTransferredFromOtherHealthFacilityWorkflowState()
+            .getProgramWorkflowStateId());
+    map.put(
+        "adultoSeguimentoEncounterType",
+        hivMetadata.getAdultoSeguimentoEncounterType().getEncounterTypeId());
+    map.put(
+        "pediatriaSeguimentoEncounterType",
+        hivMetadata.getPediatriaSeguimentoEncounterType().getEncounterTypeId());
+    map.put(
+        "arvPharmaciaEncounterType",
+        hivMetadata.getARVPharmaciaEncounterType().getEncounterTypeId());
+    map.put(
+        "masterCardEncounterType", hivMetadata.getMasterCardEncounterType().getEncounterTypeId());
+    map.put("arvStartDateConcept", hivMetadata.getARVStartDateConcept().getConceptId());
+    map.put("arvPlanConcept", hivMetadata.getARVPlanConcept().getConceptId());
+    map.put("startDrugs", hivMetadata.getStartDrugs().getConceptId());
+    map.put(
+        "masterCardDrugPickupEncounterType",
+        hivMetadata.getMasterCardDrugPickupEncounterType().getEncounterTypeId());
+    map.put("artPickupConcept", hivMetadata.getArtPickupConcept().getConceptId());
+    map.put("artDatePickupMasterCard", hivMetadata.getArtDatePickupMasterCard().getConceptId());
+
+    String query =
+        ""
+            + "SELECT external.patient_id "
+            + "FROM patient external "
+            + "     INNER JOIN encounter e "
+            + "        ON e.patient_id = external.patient_id "
+            + "     INNER JOIN obs o "
+            + "        ON o.encounter_id=e.encounter_id "
+            + "WHERE external.voided = 0 "
+            + "    AND e.voided = 0 "
+            + "    AND o.voided = 0 "
+            + "    AND e.encounter_type = ${prevencaoPositivaSeguimentoEncounterType} "
+            + "    AND (o.concept_id = ${memberShipPlanConcept} AND o.value_coded IN (${yesConcept},${noConcept}) "
+            + "        OR "
+            + "        o.concept_id = ${counseledOnSideEffectsOfArtConcept} AND o.value_coded IN (${yesConcept},${noConcept}) "
+            + "        OR "
+            + "        o.concept_id = ${adherenceEvaluationConcept} AND o.value_coded IN (${goodConcept}, ${arvAdherenceRiskConcept}, ${badConcept}) "
+            + "        ) "
+            + "    AND encounter_datetime "
```

Comment:
@cfaife this part of the query mas be the way you did with D1http://197.249.6.250:8080/fghmaster/module/htmlformentry/htmlForm.form?id=48

Reply:
this C1 indicator

Reply:
This works, as it asks for OR